akka/shopping-cart-scala/kubernetes/shopping-cart-service-cr.yml

```diff
@@ -1,25 +0,0 @@
-apiVersion: "v1"
-kind: "Namespace"
-metadata:
-  name: "shopping"
----
-apiVersion: akka.lightbend.com/v1
-kind: AkkaMicroservice
```

Comment:
What is / was that? Our own custom resource or so? And we do not use / maintain it anymore?